sdk/js/src/bridge/parseSequenceFromLog.ts

```diff
@@ -161,6 +161,22 @@ export function parseSequenceFromLogNear(
   return null;
 }
 
-export function parseSequenceFromLogAptos(result: Types.UserTransaction): string {
-  return result.sequence_number;
+export async function parseSequenceFromLogAptos(
+  client: AptosClient,
+  coreBridgeAddress: string,
+  hash: string
+): Promise<string | null> {
+  // TODO: this may throw, do we want to swallow errors and return null?
+  const result = (await client.waitForTransactionWithResult(
```

Comment:
we should not do this in the SDK - in the case of a UI, it is likely the caller already has the result (otherwise how would they know the tx was completed)